x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/settings/settings_context_menu/settings_context_menu.tsx

```diff
@@ -95,26 +98,37 @@ export const SettingsContextMenu: React.FC<Params> = React.memo(
       setIsResetConversationModalVisible(true);
     }, [closePopover]);
 
-    const handleNavigateToSettings = useCallback(
-      () =>
+    const handleNavigateToSettings = useCallback(() => {
+      if (assistantAvailability.isAI4SOC) {
+        navigateToApp('securitySolutionUI', {
+          deepLinkId: SecurityPageName.configurationsAiSettings,
+        });
+        showAssistantOverlay?.({ showOverlay: false });
+      } else {
         navigateToApp('management', {
           path: 'kibana/securityAiAssistantManagement',
-        }),
-      [navigateToApp]
-    );
+        });
+      }
+    }, [assistantAvailability.isAI4SOC, navigateToApp, showAssistantOverlay]);
 
     const handleNavigateToAnonymization = useCallback(() => {
       showAnonymizationModal();
       closePopover();
     }, [closePopover, showAnonymizationModal]);
```

Comment:
Would we want to have the same behaviour for anonymization? It only opens the modal at the moment:

https://github.com/user-attachments/assets/95be407a-1d40-499d-ada9-7b70ebb810ae

Reply:
yep, I only changed the link behavior. The modals should remain